index.html

```diff
@@ -57,5 +59,204 @@ <h1>
         </ul>
       </main>
     </div>
+
+    <!-- Toolbar-Headline Section Ends -->
+
+    <!-- Work Section Starts -->
+
+    <div class="all-project-section">
+      <!-- Project 1 -->
+      <div class="work-section">
+        <div class="snapshot">
+          <img src="./images/snapshot-1.svg" alt="" />
+        </div>
+        <div>
+          <p class="snapshot-title">Tonic</p>
+          <div class="client-info">
+            <p class="client">CANOPY</p>
+            <div class="circle"></div>
+            <p>Back End Dev</p>
+            <div class="circle"></div>
+            <p>2015</p>
+          </div>
+          <p class="project-content">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="used-language">
+            <li class="language">html</li>
+            <li class="language">css</li>
+            <li class="language">javascript</li>
+          </ul>
+          <div href="#" class="see-project"><a>See Project</a></div>
```

Comment:
- [Optional] Kindly consider adding a pressed and hover state to all your buttons. You can use the `:hover` and `:active` pseudo-classes.
![image](https://user-images.githubusercontent.com/94551484/221342123-1f3921c7-1281-4a8c-8107-d7c83c04b6e0.png)

Reply:
done